### PR TITLE
updated to 7.44.0.Final

### PR DESCRIPTION
**Pull request by mbiarnes**
the links to 
https://repository.jboss.org/nexus/service/local/artifact/maven/redirect?r=snapshots&g=org.optaweb.employeerostering&a=optaweb-employee-rostering-distribution&v=7.45.0-SNAPSHOT&e=zip

and 

https://repository.jboss.org/nexus/service/local/artifact/maven/redirect?r=snapshots&g=org.optaweb.vehiclerouting&a=optaweb-vehicle-routing-distribution&v=7.45.0-SNAPSHOT&e=zip

are dead somehow - like there is nothing! Please check this.

### CODE REVIEW

**Comment by ge0ffrey** [comment]
We can't merge this if those links are broken. Jiri's investigating.

**Comment by mbiarnes** [comment]
@ge0ffrey @yurloc I am not sure, but I would say the nightly artifacts of 7.45.0-SNAPSHOT are not uploaded because the don't exist. For this a deploy job has to run for the 7.x branches. But right now we only had them for master. Deplo jobs for 7.x branch were added https://rhba-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/KIE/job/master/job/deployedRep/ but they fail. When these works the 7.45.0-SNAPSHOTs will be available. (I would say)

**Comment by mbiarnes** [comment]
@ge0ffrey @yurloc both links work now:

https://repository.jboss.org/nexus/service/local/artifact/maven/redirect?r=snapshots&g=org.optaweb.employeerostering&a=optaweb-employee-rostering-distribution&v=7.45.0-SNAPSHOT&e=zip

https://repository.jboss.org/nexus/service/local/artifact/maven/redirect?r=snapshots&g=org.optaweb.vehiclerouting&a=optaweb-vehicle-routing-distribution&v=7.45.0-SNAPSHOT&e=zip

**Comment by yurloc** [comment]
Awesome! Thanks @mbiarnes :slightly_smiling_face:

**Review by yurloc — APPROVED** [review]

**Pull request merged**
